_posts/2020-11-09-the-cloud-data-warehouse.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: post
+title: The Cloud Data Warehouse - How Do You Like Yours?
+date: 2020-11-09
+tags: cloud data warehouse snowflake bigquery redshift
+image: img/blog/sos/snowcdw/banner.jpg
+author: Paul Symons
+---
+<center><img src="/img/blog/snowcdw/banner.jpg" /></center><br/>
+
+## The Modern Analytics Engineering Stack
+
+I recently took a little time to read through [The Analytics Guide Book](https://www.holistics.io/books/setup-analytics/start-here-introduction/) by  [Holistics](https://www.holistics.io/product/). It resonates a lot with the way we think about working with data at [Mechanical Rock](https://mechanicalrock.io/), especially on the following principles:
+
+* ELT over ETL
+* SQL based analytics is the way forward
+* Use Technology To Replace Labor Whenever Possible (*So your people can focus on business value instead*)
+
+It's rare to see a book presented in such an accessible way that manages to balance complexity and insight - I recommend it to anyone looking to re-evaluate how they maximise return on investment from their data.
+
+This blog post is in two parts; in this article we will elaborate on modern Cloud Data Warehouses; in the second part we will focus on how their features allow a more flexible and agile way of delivering analytics engineering, as described in the Analytics Guide Book.
+
+## The Cloud Data Warehouse
+
+
+A Data Warehouse is an **enabling technology** for your business to **make decisions** in relation to **the data you gather**. By arranging, conforming, and curating your data to radiate the key performance indicators of your business, a Data Warehouse forms one half of *the presentation layer* - alongside visualisation tools - that enable users in your organisation to identify insights. At least, this is the **traditional view**.
+
+Modern Cloud Data Warehouses can support the existing status quo, including various data modelling methodologies such as Kimball, Inmon or Data Vault for example. Additionally, sticking the word ***cloud*** in front of *data warehouse* does what you would expect cloud to do, in addressing many of the technological limitations or complications associated with traditional Data Warehouses - namely storage availability, compute scalability, and high availability.
+
+However, modern Cloud Data Warehouse offerings typically go far beyond that. Whilst delivering the core competencies of what a Data Warehouse should offer, they often extend these capabilities with features not readily seen before. For example,
```

Comment:
`For example:` (with a colon instead of a comma)

Reply:
"Additionally" sentence is convoluted. 

Suggest "Sticking the word ***cloud*** in front of *data warehouse* does what you would expect cloud to do – it addresses many of the technological limitations associated with traditional Data Warehouses, namely storage availability, compute scalability and high availability.

Reply:
Also "However" sentence has too many conditional words. 

How about "However, modern Cloud Data Warehouses typically go far beyond that. Whilst delivering the core competencies of a Data Warehouse, they extend these capabilities with features not readily seen before."

Reply:
> "Additionally" sentence is convoluted.
> 
> Suggest "Sticking the word _**cloud**_ in front of _data warehouse_ does what you would expect cloud to do – it addresses many of the technological limitations associated with traditional Data Warehouses, namely storage availability, compute scalability and high availability.

yeah... i had added complications because while data warehouses typically do support high availability / clustering, it's manual and specialised to manage things like clustering replication and failover. but i'll simplify it